### DISCUSSION TITLE
Add inches of water column to pressure units

### DISCUSSION THREAD

**Discussion by jaminh** [home-assistant/architecture#764] [Entity Models] [answered]
Add inches of water column to the list of valid pressure units. Inches of water column is a fairly common unit of pressure in the US HVAC industry, particularly for measuring static pressure of duct systems.

**Comment by jaminh** [comment]
I would like to bring up this issue again, it is at least tangentially related to https://github.com/home-assistant/architecture/discussions/839.

**Comment by emontnemery** [comment]
Can you please give some examples of when this unit would be useful in a home automation context?

**Reply by jaminh** [reply]
It is admittedly a pretty niche use case, but it is the standard unit for measuring the static pressure of ducts in residential HVAC systems, in the US at least. I made a sensor to measure this along with supply/return side temperature/relative humidity to try to provide more insights into the performance of my HVAC system (more details here https://community.home-assistant.io/t/more-hvac-monitoring/443415). For the most part everything is working, I just get an error when HA is starting up: 2023-02-05 07:05:59.384 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.hvac_monitor_pressure (<class 'homeassistant.components.esphome.sensor.EsphomeSensor'>) is using native unit of measurement 'inH2O' which is not a valid unit for the device class ('pressure') it is using; expected one of ['Pa', 'inHg', 'bar', 'mmHg', 'hPa', 'kPa', 'cbar', 'psi', 'mbar']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+esphome%22. So for the most part I am just trying to get rid of that error.

**Reply by epenet** [reply]
If that is the only reason, then the easiest is to remove the `pressure` device class from the sensor...

**Comment by frenck** [comment, marked as answer]
We have discussed this today and decided not to add this at this point.

The main reason is it is niche and we have no integrations using it at this point (besides the ones that provide custom units, which are left out of scope for these cases).

../Frenck
